migration-tools.md

```diff
@@ -13,34 +13,83 @@ This document introduces the user scenarios, advantages, and limitations of thes
+
+## [Dumpling](/dumpling-overview.md)
+
+| User scenario | <span style="font-weight:normal">Full data export from MySQL or TiDB</span> |
+|---|---|
+| **Upstream (or the imported source file)** | MySQL, TiDB |
+| **Downstream (or the output file)** | SQL, CSV |
+| **Advantages** | <ul><li>Support the table-filter feature that enables you to filter data easier</li><li>Support exporting data to Amazon S3</li></ul> |
+| **Limitation** | <ul><li>If you want to restore the exported data to a database other than TiDB, it is recommended to use Dumpling.</li><li>If you want to restore the exported data to another TiDB cluster, it is recommended to use Backup & Restore (BR).</li></ul> |
+
+## [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)
+
+| User scenario | <span style="font-weight:normal">Full data import into TiDB</span> |
+|---|---|
+| **Upstream (or the imported source file)** | <ul><li>Files exported from Dumpling</li><li>Parquet files exported by Amazon Aurora or Apache Hive</li><li>CSV files</li><li>Data read from local disks or Amazon S3</li></ul> |
+| **Downstream (or the output file)** | TiDB |
+| **Advantages** | <ul><li>Support quickly importing a large amount of data and quickly initializing a specific table in a TiDB cluster </li><li>Support checkpoints to store the import progress, so that `tidb-lightning` continues importing from where it lefts off after restarting</li><li>Support data filtering</li></ul> |
+| **Limitation** | <ul><li>If Local-backend is used for data import, during the import process, the TiDB cluster cannot provide services.</li><li> If you do not want the TiDB services to be impacted, perform the data import according to TiDB Lightning TiDB-backend.</li></ul> |
+
+## [Backup & Restore (BR)](/br/backup-and-restore-overview.md)
+
+| User scenario | <span style="font-weight:normal">Migrate a large amount of TiDB custer data by backing up and restoring data</span> |
+|---|---|
+| **Upstream (or the imported source file)** | TiDB |
+| **Downstream (or the output file)** | SST, backup.meta files, backup.lock files |
+| **Advantages** |<ul><li>Suitable for migrating data to another TiDB cluster</li><li>Support backing up data to an external storage for disaster recovery</li></ul> |
+| **Limitation** |<ul><li>When BR restores data to the upstream cluster of TiCDC or Drainer, the restored data cannot be replicated to the downstream by TiCDC or Drainer.</li><li>BR supports operations only between clusters that have the same `new_collations_enabled_on_first_bootstrap` value.</li></ul> |
+
+## [TiCDC](/ticdc/ticdc-overview.md)
+
+| User scenario | <span style="font-weight:normal">This tool is implemented by pulling TiKV change logs. It can restore data to a consistent state with any upstream TSO, and support other systems to subscribe to data changes.</span> |
```

Comment:
```suggestion
| User scenario | <span style="font-weight:normal">This tool is implemented by pulling TiKV change logs. It can restore cluster data to a consistent state with any upstream TSO, and support other systems to subscribe to data changes.</span> |
```